recipes/geos/all/patches/fix-cmake-3.9.0.patch

```diff
@@ -0,0 +1,41 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -19,7 +19,6 @@ set(DEFAULT_BUILD_TYPE Release)
+ # Require CMake 3.13+ with VS generator for complete support of VS versions
+ # and support by AppVeyor.
+ if(${CMAKE_GENERATOR} MATCHES "Visual Studio")
+-  cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
```

Comment:
is this required?

Reply:
It builds with Visual Studio and lower CMake versions. It's seem to be required for their CI only. It was already removed in prevous versions.